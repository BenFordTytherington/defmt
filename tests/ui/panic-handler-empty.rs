#![no_main]
#![no_std]

#[panic_handler]
fn foo() {}
