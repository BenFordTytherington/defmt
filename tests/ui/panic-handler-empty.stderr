error: language item required, but not found: `eh_personality`

error: return type should be `!`
 --> $DIR/panic-handler-empty.rs:5:10
  |
5 | fn foo() {}
  |          ^

error: function should have one argument
 --> $DIR/panic-handler-empty.rs:5:1
  |
5 | fn foo() {}
  | ^^^^^^^^
